.travis/install_ctverif.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#  http://aws.amazon.com/apache2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+#
+
+set -e
+
+usage() {
+    echo "install_ctverif.sh install_dir"
+    exit 1
+}
+
+if [ "$#" -ne "1" ]; then
+    usage
+fi
+
+INSTALL_DIR=$1
+
+cd $INSTALL_DIR
+
+#install smack
+git clone --depth 1 https://github.com/smackers/smack.git -b develop
+cd smack/bin
+./build.sh
+
+source $INSTALL_DIR/smack.environment
+
+#install ctverif
+cd $INSTALL_DIR
+git clone --depth 1 https://github.com/danielsn/verifying-constant-time.git -b s2n-test
```

Comment:
I'd prefer if we cloned the [official repo](https://github.com/imdea-software/verifying-constant-time), but see that you [have a couple of commits that aren't in the official repo](https://github.com/imdea-software/verifying-constant-time/compare/test-automation...danielsn:s2n-test). 

Were we planning on maintaining our own copy of `verifying-constant-time` or were we planning on getting your changes merged into their repo?

Also, there are a couple of files in the ` examples/s2n-travis` directory in the other repo. I think it would be better if those files were stored in this repo (something like`./tests/ctverif` or `./tests/const-time` directory), that way if we need to make any changes we won't be blocked on getting our changes merged in a different different repo. 

From first looking at this PR, I didn't realize that you were actually overriding the `s2n_annotations.h` with another file stored in your repo.